deployer/README.md

```diff
@@ -1175,3 +1175,260 @@ the first created. You can check if the route in question is defined in multiple
+      @include configs.d/openshift/filter-kibana-transform.conf
+      @include configs.d/openshift/filter-k8s-flatten-hash.conf
+      @include configs.d/openshift/filter-k8s-record-transform.conf
+      @include configs.d/openshift/filter-syslog-record-transform.conf
+      @include configs.d/openshift/filter-common-data-model.conf
+      @include configs.d/openshift/filter-post-*.conf
+    ##
+
+    ## matches
+      @include configs.d/openshift/output-operations.conf
+      @include configs.d/openshift/output-applications.conf
+    ##
+    </label>
+
+Then, create the configmap that will be used for the Enriching Fluentd, using the
+file created in the previous step as the source for the configmap:
+
+    $ oc create configmap logging-fluentd-enrichment \
+        --from-file=fluent.conf=enrichment-fluent.conf
+
+The next step will be to create the Deployment configuration that will be used to
+scale up the enrichment fluentd pods. Save the following file to your local file
+system as `enrichment-fluentd-dc.yaml`. Be sure to update the value of "K8S_HOST_URL"
+and "OPS_HOST" if necessary:
+
+    apiVersion: v1
+    kind: "DeploymentConfig"
+    metadata:
+      name: "logging-enrichment-fluentd"
+      labels:
+        provider: openshift
+        component: "fluentd"
+    spec:
+      replicas: 0
+      selector:
+        provider: openshift
+        component: "fluentd"
+      triggers: {}
+      strategy:
+        resources: {}
+        rollingParams:
+          intervalSeconds: 1
+          timeoutSeconds: 600
+          updatePeriodSeconds: 1
+        type: Recreate
+      template:
+        metadata:
+          name: enrichment-fluentd-elasticsearch
+          labels:
+            provider: openshift
+            component: "fluentd"
+        spec:
+          terminationGracePeriodSeconds: 300
+          serviceAccountName: aggregated-logging-fluentd
+          containers:
+          - name: enrichment-fluentd-elasticsearch
+            image: ${IMAGE_PREFIX}logging-fluentd:${IMAGE_VERSION}
+            imagePullPolicy: Always
+            securityContext:
+              privileged: true
+            resources:
+              limits:
+                cpu: 100m
+            ports:
+              - containerPort: 1095
+                hostPort: 1095
+                protocol: TCP
+                purpose: "to prevent more than one per node"
+                volumeMounts:
+                - name: runlogjournal
+                  mountPath: /run/log/journal
+                - name: varlog
+                  mountPath: /var/log
+                - name: varlibdockercontainers
+                  mountPath: /var/lib/docker/containers
+                  readOnly: true
+                - name: config
+                  mountPath: /etc/fluent/configs.d/user
+                  readOnly: true
+                - name: certs
+                  mountPath: /etc/fluent/keys
+                  readOnly: true
+                - name: dockerhostname
```

Comment:
I don't know if all of this hostPath stuff is needed. My guess is that they are not really needed